websites/R/Rádio Animu/dist/metadata.json

```diff
@@ -1,27 +1,33 @@
 {
   "author": {
-    "name": "Bas950",
-    "id": "241278257335500811"
+    "name": "bossatossa",
+    "id": "514756351327731729"
   },
   "contributors": [
     {
       "name": "fror",
       "id": "515668127829458945"
-    }
+    },
+	{
+	 "name": "Bas950",
+     "id": "241278257335500811"
+	}
   ],
   "service": "Rádio Animu",
   "description": {
     "en": "The most moe radio in Brazil",
     "pt_BR": "A rádio mais moe do Brasil"
   },
   "url": "www.animu.com.br",
-  "version": "1.2.5",
-  "logo": "https://i.imgur.com/vMsQ4Bd.png",
-  "thumbnail": "https://i.imgur.com/Ke5HBcL.png",
+  "version": "1.2.6",
+  "logo": "https://i.imgur.com/SHAH0WR.png",
+  "thumbnail": "https://i.imgur.com/YS3pllQ.png",
   "color": "#2c004b",
   "tags": [
     "brasil",
-    "anime"
+    "anime",
+	"radio",
+	"moe"
```

Comment:
fix the indentation here too

Reply:
thanks for the suggestions 👍🏼 
applied them on https://github.com/PreMiD/Presences/pull/3430/commits/999f86f0c9d30287ba49744aa6aaf7a12d89e700